tests/lambda_functions/vetext_incoming_forwarder_lambda/test_vetext_incoming_forwarder_lambda.py

```diff
@@ -118,25 +126,16 @@ def test_failed_sqs_invocation_call_throws_general_exception(mocker, event):
 
 @pytest.mark.parametrize('event', [(albInvokedWithoutAddOn), (albInvokeWithAddOn)])
 def test_failed_alb_invocation_call_throws_general_exception(mocker, event):
```

Comment:
How is this function directly testing that HttpException was raised?

Reply:
if exception wasnt raised then retry sqs wouldnt be calle